package-templates/rust/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+pub use {{project-name}};
+pub use {{project-name}}-plugin;
```

Comment:
This `pub use *-plugin` line should be removed. I've seen this in other packages, but it's wrong in those, too.